### PR TITLE
core: constant subjects for notifications

### PR DESCRIPTION
**Pull request by buck54321**
Broken off from #717. Includes `MatchNote` changes that aren't yet used, and a correction for the details of an `OrderNote`.

### CODE REVIEW

**Comment by chappjc** [comment]
@dajohi Please have a look at the constants and how they are used when you have a chance https://github.com/decred/dcrdex/pull/818/files#diff-c5fccd8f7f5c11f3e53e2cf6cd8e633e9899ca29b7a20a30df549739dfcb5964R169

File: client/core/notification.go [hunk 1]
```diff
@@ -148,6 +166,38 @@ type OrderNote struct {
...
+	MatchesRefundedSubject      = "Matches Refunded"
+	MatchRevokedSubject         = "Match revoked"
+	RevokeSubject               = "revoke"
+	MatchRecoveredSubject       = "Match recovered"
+	CancellingOrderSubject      = "Cancelling order"
+	OrderStatusUpdateSubject    = "Order status update"
+	MatchResolutionErrorSubject = "Match resolution error"
+	FailedCancelSubject         = "Failed cancel"
+)
+
```
Comment (dajohi) [inline client/core/notification.go:200]: I'd prefer "Subject" be the prefix instead.

**Review by dajohi — DISMISSED** [review]

File: client/core/notification.go (hunk 1 above)
Comment (chappjc) [reply client/core/notification.go:200]: Second that.

Comment (buck54321) [reply client/core/notification.go:200]: Done

File: client/core/notification.go [hunk 2]
```diff
@@ -163,6 +174,11 @@ type WithdrawNote struct {
 	db.Notification
 }
 
+const (
+	WithdrawErrorSubject = "Withdraw error"
+	WithdrawSendSubject  = "Withdraw sent"
```
Comment (chappjc) [inline client/core/notification.go:179]: Missed the subject prefix on these two.

**Review by chappjc — CHANGES_REQUESTED** [review]
Looks good, just two subject variable names need adjusting.

**Review by chappjc — COMMENTED** [review]
Actually, I'm getting the following for new matches:

image

**Review by chappjc — APPROVED** [review]

**Pull request merged**